browser/resources/side_panel/bookmarks/brave_overrides/power_bookmarks_list.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { RegisterPolymerPrototypeModification } from 'chrome://resources/brave/polymer_overriding.js'
+
+RegisterPolymerPrototypeModification({
+    'power-bookmarks-list': prototype => {
+        const originalOnBookmarkMoved = prototype.onBookmarkMoved
+        prototype.onBookmarkMoved = function (
+            bookmark: chrome.bookmarks.BookmarkTreeNode,
+            oldParent: chrome.bookmarks.BookmarkTreeNode,
+            newParent: chrome.bookmarks.BookmarkTreeNode) {
+            originalOnBookmarkMoved.apply(this, bookmark, oldParent, newParent)
```

Comment:
@fallaciousreasoning I'm seeing that passed `bookmark, oldParent, newParent` args become `undefined` in `originalOnBookmarkMoved`. They're definitely valid value in this method but all undefined in original method.
I'm investigating this but still no clue. need your help :)

Reply:
OMG... my bad. I should use `call` instead of `apply` :) NVM!